### PR TITLE
About page

### PR DESCRIPTION
**Pull request by lucaaeberhard**
Ich habe die About-Page bis auf weiteres fertiggestellt.

### CODE REVIEW

File: app/Http/Controllers/aboutController.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+use Illuminate\Http\Request;
+
+class aboutController extends Controller
+{
+    public function index()
+    {
+    return view('about');
+    }
+}
+
```
Comment (lucaaeberhard) [inline app/Http/Controllers/aboutController.php:14]: Added About-Controller routing for the about-page is now working over this controller

File: resources/views/about.blade.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <svg class="bd-placeholder-img card-img-top" width="100%" height="225" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid slice" focusable="false" role="img" aria-label="Placeholder: Thumbnail"><title>Umfrage ausfüllen</title><rect width="100%" height="100%" fill="#55595c"></rect><text x="35%" y="50%" fill="#eceeef" dy=".3em">Umfrage ausfüllen</text></svg>
+            <div class="card-body">
+              <p class="card-text">Wenn Sie einen Pin oder einen Link erhalten können Sie ganz einfach mit diesem an der Umfrage teilnehmen. Klicken SIe dafür einfach den Link an oder geben Sie den erhaltenen Pin ein</p>
+            </div>
+          </div>
+        </div>
+     </div>   
+
+        
+@endsection
```
Comment (lucaaeberhard) [inline resources/views/about.blade.php:41]: Added a new About-page design and text

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -7,24 +7,24 @@
     <!-- CSRF Token -->
     <meta name="csrf-token" content="{{ csrf_token() }}">
 
-    <title>{{ config('app.name', 'Laravel') }}</title>
+    <title>{{ config('app.name', 'Elameter') }}</title>
```
Comment (lucaaeberhard) [inline resources/views/layouts/app.blade.php:10]: Changed app.name to Elameter

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -7,24 +7,24 @@
...
 
     <!-- Scripts -->
     <script src="{{ asset('js/app.js') }}" defer></script>
 
     <!-- Fonts -->
-    <link rel="dns-prefetch" href="//fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,600;1,400&display=swap" rel="stylesheet">
```
Comment (lucaaeberhard) [inline resources/views/layouts/app.blade.php:16]: Changed font

File: resources/views/layouts/app.blade.php [hunk 5]
```diff
@@ -7,24 +7,24 @@
...
-<body>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+</head> 
+<body class="px-0">
     <div id="app">
         <nav class="navbar navbar-expand-md navbar-light bg-white shadow-sm">
             <div class="container">
                 <a class="navbar-brand" href="{{ url('/') }}">
-                    {{ config('app.name', 'Laravel') }}
+                    {{ config('app.name', 'Elameter') }}
```
Comment (lucaaeberhard) [inline resources/views/layouts/app.blade.php:27]: Changed app.name to Elameter

File: resources/views/layouts/app.blade.php [hunk 6]
```diff
@@ -7,24 +7,24 @@
...
-    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,600;1,400&display=swap" rel="stylesheet">
 
     <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
-</head>
-<body>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+</head> 
+<body class="px-0">
```
Comment (lucaaeberhard) [inline resources/views/layouts/app.blade.php:22]: Added bootstrap-stylesheet

File: resources/views/layouts/app.blade.php [hunk 7]
```diff
@@ -76,5 +76,8 @@
             @yield('content')
         </main>
     </div>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```
Comment (lucaaeberhard) [inline resources/views/layouts/app.blade.php:81]: Added bootstrap javascript

File: resources/views/welcome.blade.php [hunk 8]
```diff
@@ -85,7 +85,7 @@
                 </div>
 
                 <div class="links">
-                    <a href="">About</a>
+                    <a href="/about">About</a>
```
Comment (lucaaeberhard) [inline resources/views/welcome.blade.php:88]: Linked about-page

File: routes/web.php [hunk 9]
```diff
@@ -17,6 +17,8 @@
     return view('welcome');
 });
 
+Route::get('/about', 'aboutController@index')->name('about');
+
```
Comment (lucaaeberhard) [inline routes/web.php:21]: Forwards "/about" - requests to the index-method in the aboutController

**Comment by DeNic0la** [comment]
I Viewed the Code, i think its okey

**Pull request merged**